src/components/MainLayout/MainLayout.jsx

```diff
@@ -1,50 +1,52 @@
-import { Suspense } from 'react';
-import { Outlet, useLocation } from 'react-router-dom';
+import {Suspense} from 'react';
```

Comment:
Check your prettier and make to reformat your files again

Reply:
update settings Prettier